examples/neuralop/checkpoint_FNO_darcy.py

```diff
@@ -0,0 +1,145 @@
+"""
+Training a TFNO on Darcy-Flow
+=============================
+
+In this example, we demonstrate how to use the small Darcy-Flow example we ship with the package
+to train a Tensorized Fourier-Neural Operator
+"""
+
+import sys
+
+# %%
+#
+import paddle
+
+from ppsci.contrib.neuralop import H1Loss
+from ppsci.contrib.neuralop import LpLoss
+from ppsci.contrib.neuralop import Trainer
+from ppsci.contrib.neuralop.datasets import load_darcy_flow_small
+from ppsci.contrib.neuralop.models import TFNO
+from ppsci.contrib.neuralop.training import CheckpointCallback
+from ppsci.contrib.neuralop.utils import count_model_params
+
+device = "cpu"
+
+
+# %%
+# Loading the Navier-Stokes dataset in 128x128 resolution
+train_loader, test_loaders, data_processor = load_darcy_flow_small(
+    n_train=1000,
+    batch_size=32,
+    test_resolutions=[16, 32],
+    n_tests=[100, 50],
+    test_batch_sizes=[32, 32],
+)
+
+
+# %%
+# We create a tensorized FNO model
+
+model = TFNO(
+    n_modes=(16, 16),
+    hidden_channels=32,
+    projection_channels=64,
+    factorization="tucker",
+    rank=0.42,
+)
+model = model.to(device)
+
+n_params = count_model_params(model)
+print(f"\nOur model has {n_params} parameters.")
+sys.stdout.flush()
+
+
+# %%
+# Create the optimizer
+optimizer = paddle.optimizer.Adam(
+    learning_rate=8e-3, parameters=model.parameters(), weight_decay=1e-4
+)
+scheduler = paddle.optimizer.lr.CosineAnnealingDecay(
+    learning_rate=optimizer.get_lr(), T_max=30
+)
+optimizer.set_lr_scheduler(scheduler=scheduler)
```

Comment:
optimizer和lr_scheduler建议按照下面的代码进行设置。

```suggestion
scheduler = paddle.optimizer.lr.CosineAnnealingDecay(
    learning_rate=8e-3, T_max=30
)
optimizer = paddle.optimizer.Adam(
    learning_rate=scheduler, parameters=model.parameters(), weight_decay=1e-4
)
```

Reply:
https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.lr_scheduler.CosineAnnealingLR.html#zhuanxieshili
这里我是参考转写示例写的